### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by lu-jim**
Third PR for the week.
## Changes
- Added client-side validation
- Customized error message

### CODE REVIEW

File: formValidation.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const alertLowercase = formEmail.appendChild(createNode('label', 'email-error'));
+alertLowercase.innerHTML = 'Error: Please use lowercase letters for your email';
+
+formStatus.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (formEmail.value === formEmail.value.toLowerCase()) {
+    formStatus.submit();
+  } else {
+    const emailAlert = formEmail.insertAdjacentElement('afterend', createNode('p', 'email-error'));
+    emailAlert.innerHTML = 'Error: Please use lowercase letters for your email';
```
Comment (oxenprogrammer) [inline formValidation.js:18]: **Changes Required**
- [ ] Good effort on implementing the error message when the email isn't all lower case. However, this message should rather show up close to the submit button according to the specs. Let us try and move it close to the submit button from its current position.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @lu-jim, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: formValidation.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+function createNode(type, nodeClass) {
+  const node = document.createElement(type);
+  if (nodeClass) node.className = nodeClass;
+  return node;
+}
+
+const formStatus = document.getElementById('fs-frm');
+const formEmail = document.getElementById('email-address');
+const formButton = document.getElementById('form-submit-button');
+const alertLowercase = formButton.appendChild(createNode('label', 'email-error'));
```
Comment (Alexoid1) [inline formValidation.js:10]: - [ ] Good job implementing the error message but now another issue has appeared. Well, when the user clicks on the submit button will append a new error message and the error message will be shown several times kindly fix that in order to make the error message appear just one time.
 Captura de pantalla_2021-10-15_20-10-10

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @lu-jim,

Impressive project!
You are doing a great job so far, and the project is looking good.
- No linter errors.
- Descriptive README.MD file.
- Follow Git-hub flow.

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes :recycle:
For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Alexoid1 — APPROVED** [review]
# STATUS: APPROVED!!!
## Hi @lu-jim  Great job! 
## Congratulations :tada: Thanks for implementing changes suggested.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

brent-rambo-thumbs-up


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**